docs/reference/matchTag.md

```diff
@@ -0,0 +1,13 @@
+# Setting: matchTag
+
+`matchTag TAGNAME` can be defined as [CLI argument](./define-settings.md#cli) or
+[conversion comment](./define-settings.md#conversion).
```

Comment:
This is incorrect, it's a method comment that is inheritable. Use the text from https://goverter.jmattheis.de/reference/matchIgnoreCase

Reply:
Thanks, this was something I was definitely not sure about.